app/src/main/java/com/example/dndwiki/repository/SpellRepository.kt

```diff
@@ -51,4 +43,35 @@ class SpellRepository(
             repoStatus = false
         }
     }
+
+    private fun isAlreadySavedToday(): Boolean {
```

Comment:
Nice method 👍 

But we can make it better :) 

Watch this presentation: https://www.youtube.com/watch?v=2a_ytyt9sf8&list=WL&index=9&ab_channel=UnityCoin 

And afterwards come back to this code and see if you can find some comments that could be removed by having better names insetad :)